docs/plugins.md

```diff
@@ -66,35 +66,15 @@ For a complete example, see:
 
 ### Gradle
 
-Gradle does not
-([yet](https://github.com/gradle/gradle/blob/master/design-docs/java-annotation-processing.md))
-have built-in support for setting the processor path, but its flexibility makes
-it relatively easy to do manually:
+Starting in version 4.6, Gradle provides support for [configuring the processor
+path](https://docs.gradle.org/4.6/release-notes.html#convenient-declaration-of-annotation-processor-dependencies):
 
 ```gradle
-configurations {
-  annotationProcessor
-}
-
 dependencies {
   annotationProcessor project(':custom-checks')
 }
-
-tasks.withType(JavaCompile) {
-  options.compilerArgs += [ '-processorpath', configurations.annotationProcessor.asPath ]
-}
 ```
 
-Gradle plugins exist that also offer this configurability:
-[`net.ltgt.apt`](https://plugins.gradle.org/plugin/net.ltgt.apt) for standard
-Java projects, or
-[`android-apt`](https://bitbucket.org/hvisser/android-apt) or the
-[experimental new Android toolchain](https://sites.google.com/a/android.com/tools/tech-docs/jackandjill)
-for Android projects.
-
-For a complete example using the `net.ltgt.apt` plugin, see:
-[examples/plugin/gradle](https://github.com/google/error-prone/tree/master/examples/plugin/gradle).
```

Comment:
This should probably be kept (and the example updated)

Reply:
I think that happened in https://github.com/google/error-prone/commit/250258827f9d1b36f5d08e87475c9d8880521717 (very belated thanks!). I edited this to restore the pointer to the example.